scout/adapter/mongo/query.py

```diff
@@ -296,7 +295,7 @@ def build_query(
             secondary_filter = self.secondary_query(query, mongo_query)
             # If there are no primary criteria given, all secondary criteria are added as a
             # top level '$and' to the query.
-            if primary_terms is False:
+            if secondary_filter and primary_terms is False:
```

Comment:
💯

Reply:
Should be fine, as there is already a check for existing `$and` in the coordinate query below.